src/apps/backend/modules/task/internal/store/task-db.ts

```diff
@@ -6,6 +6,7 @@ export interface TaskDB {
   active: boolean;
   description: string;
   title: string;
+  sharedWith: Types.ObjectId[];
```

Comment:
This is quite poor programming practice:

1. It is not advisable to store array like this especially when it can grow beyond 4-5 items
2. It makes it hard to reason with - especially when we have to query
3. This is semantically not right. SharedWith is not a property of Task. Its rather a relationship.